services/feed-manager-service/feed-manager-controller/src/main/java/com/thinkbiganalytics/feedmgr/service/feed/FeedModelTransform.java

```diff
@@ -388,6 +405,21 @@ public FeedSummary domainToFeedSummary(@Nonnull final Feed feedManagerFeed) {
     public List<FeedSummary> domainToFeedSummary(@Nonnull final Collection<? extends Feed> domain) {
         return domain.stream().map(this::domainToFeedSummary).filter(feedSummary -> feedSummary != null).collect(Collectors.toList());
     }
+    
+    /**
+     * Transforms the specified Metadata feeds to Feed Manager feed summaries.
+     *
+     * @param domain the Metadata feed
+     * @return the Feed Manager feed summaries
+     */
+    @Nonnull
+    public Page<UIFeed> domainToFeedSummary(@Nonnull final Page<Feed> domain, @Nonnull final Pageable pageable) {
+        List<UIFeed> summaries = domain.getContent().stream()
+                        .map(this::domainToFeedSummary)
+                        .filter(feedSummary -> feedSummary != null)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with method reference 'Objects::nonNull'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)